docs/integration/alerts-alertops-integration.md

```diff
@@ -21,5 +21,17 @@ Once the test message is visible, click the **Save Notification Hook** button to
 
 ![AlertOps Alerts Integration](https://sematext.com/docs/images/integrations/alertops-integration.png "AlertOps Integration")
 
+
+## **Mapping priority in AlertOps**
+Sematext Alerts let you set the priority for your alert rules. With this functionality, you will be able to differentiate between info, warn, error, and critical alerts. By default, all existing alert rules will have the WARN priority.
```

Comment:
> By default, all existing alert rules will have the WARN priority.

This is really something that is for existing users.  Not something we need to mention in AlertOps docs, IMHO.